README.md

```diff
@@ -103,8 +117,14 @@ Your query here
 
 4. Three supervisors with the most amount of employees they are in charge.
 
-```
-Your query here
+```sql
+SELECT e1.id as supervisor_id, count(e2.id)
```

Comment:
Instead of using `id`, you could use the supervisor_id and it's not necessary to use an inner join (keep it simple) and a where clause where `supervisor_id` is not null